frontend/components/Dashboard/Settings/DetailsForm.tsx

```diff
@@ -112,29 +109,27 @@ const DetailsForm: React.FC<DetailsFormProps> = ({ currentUser }) => {
               />
               <div className="details-form-field">
                 <label className="settings-label" htmlFor="handle">
-                  Display Name
+                  {t('profile.details.displayNameLabel')}
                 </label>
                 <input
                   type="text"
                   name="handle"
-                  value={handle}
+                  value={currentUser.handle}
```

Comment:
Since this field is disabled, but we still have it wired up to be able to change the value with `setHandle`, I removed the `useState` for handle in case someone opens the dev tools and removes the `disabled` attribute from the input element